lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -822,15 +1020,15 @@ def _set_scrape_job_spec(self, event):
                     {"groups": alert_groups}
                 )
 
-    def _set_unit_ip(self, event):
+    def _set_unit_ip(self, _: EventBase):
```

Comment:
Hmm. Rather silly to add type annotation for an unused argument. :-). Feels like religion.

Reply:
I saw type hints everywhere, so went with "Why not"